sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientBuilder.java

```diff
@@ -438,4 +453,20 @@ public KeyEncryptionKeyClientBuilder disableChallengeResourceVerification() {
 
         return this;
     }
+
+    /**
+     * Disables the ability to perform cryptographic operations locally, performing all cryptographic operations on the
+     * service side instead.
+     *
+     * <p>This method will have no effect if
+     * {@link KeyEncryptionKeyClientBuilder#buildAsyncKeyEncryptionKey(JsonWebKey)} or
+     * {@link KeyEncryptionKeyClientBuilder#buildKeyEncryptionKey(JsonWebKey)} are used to create a client.</p>
```

Comment:
Same here - do not ignore silently. Throw runtime exception in `buildClient` method is we have conflicting configuration.